### ISSUE TITLE
Next focus not working

### ISSUE THREAD

**Issue by syJSdev** [suweya/react-verification-code-input#42]
If you enter the same value for each input, the cursor isn't moved to the next input.

Here is the video example.

![1](https://user-images.githubusercontent.com/51244659/100309775-a0304b80-2f79-11eb-9e94-0b3b2aef6844.gif)

I think it is caused by the onChange event.
onChange event isn't fired when input the same value.

**Comment by suweya** [comment]
yes i also think it is caused by the onChange not trigged if enter the same value
